### ISSUE TITLE
Package in some other package mangers?

### ISSUE THREAD

**Issue by shonfeder** [realworldocaml/mdx#296]
Because I think mdx could provide killer functionality to a wide community, it could be valuable to package it for some other distribution channels! These might include

- apt
- homebrew
- arch

I'd be happy to help with one or more of these (probably beginning with apt) if it's deemed to be a good idea.

**Comment by Leonidas-from-XIV** [comment]
@shonfeder I agree it would be neat to have the binary available for users. Last time I checked though the Debian packaging guidelines said not to add Debian-specific files (e.g. a `debian/`) folder to upstream projects. So it would be up for some Debian maintainers to package it.

Getting things submitted to Homebrew and Arch Linux AUR is easier though, since that is open to essentially anyone that has access to Git(Hub).
